spec/UserDbSpec.js

```diff
@@ -2,20 +2,47 @@
 /* global it */
```

Comment:
This doesn't pass the linter. I am betting other things fail too (I won't bother leaving a bunch of comments, just one as a note)

Reply:
Done!